src/modules/market/components/market-outcome-charts--candlestick-period-selector/market-outcome-charts--candlestick-period-selector.styles.less

```diff
@@ -40,13 +40,13 @@
 
   &:first-of-type {
     h1 {
-      margin: 12px 0 8px 16px;
+      margin: 1em 0 0.75em 1.33em;
```

Comment:
@stephensprinkle Why did you intermix rem and em? I don't have a feel for this yet and would love to know what steps you take in making that decision.

Reply:
What I use to determine which unit I’ll use is: “In relation to what element should this size be derived?”

If it’s local to that element, then it’s `em`, if it doesn’t matter or is relative to the view, then it’s `rem`.

In this specific exactly, we use `em` because say we want to later change the `h1` font size, as a result, we’ll probably want to also change the margins around it.  By setting the margins relative to the sizing of the `h1`, we can scale proportionally to the change.

Reply:
Great feedback. Thanks!

Reply:
@adrake33 This might be useful insight to add to the ui conventions docs. What do you think?

Reply:
@justinbarry Yeah, that sounds like a good idea. Would you like me to add it?

(Btw, it looks like the new content on testing in https://github.com/AugurProject/docs/blob/justinbarry/ch7527/begin-writing-ui-testing-convention-docs/source/includes/_uiConventions.md has not been published to docs.augur.net. Are you waiting to publish it until you've added more info, or would you like me to publish it?)